.changeset/small-ladybugs-mate.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I think we can keep the changeset empty and when we want to publish 1.0.0 we can do a changeset?